course/src/isometry.cpp

```diff
@@ -10,9 +10,146 @@
 namespace ekumen {
 namespace math {
 
-/*
- * your code here
- */
+namespace {
+
+Vector3 translation_vector_operation(const Matrix3& matrix,
+                                     const Vector3& first_vector,
+                                     const Vector3& second_vector) {
+  Vector3 aux;
+  aux = matrix * first_vector + second_vector;
+  return aux;
+}
+
+}  // namespace
+
+
+Isometry Isometry::fromTranslation(const Vector3& r_vector) {
+  Isometry aux;
+  aux.translation_vector = r_vector;
+  return aux;
+}
+
+Vector3 Isometry::translation() const {
+  Vector3 aux{translation_vector};
+  return aux;
```

Comment:
See that you are copying the value into `aux` and then you are copying it to the output. 
Is that a reason why you did it that way?
```suggestion
  return translation_vector;
```

Reply:
Probably a habit that I caught.

Reply:
Addressed.